pkg/typechain/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@balancer-labs/typechain",
+  "version": "1.0.0",
+  "description": "Typechain bindings for Balancer Protocol V2 contracts",
+  "license": "GPL-3.0-only",
+  "homepage": "https://github.com/balancer-labs/balancer-v2-monorepo/tree/master/pkg/typechain#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/balancer-labs/balancer-v2-monorepo.git",
+    "directory": "pkg/typechain"
+  },
+  "bugs": {
+    "url": "https://github.com/balancer-labs/balancer-v2-monorepo/issues"
+  },
+  "main": "dist/index.ts",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "rm -rf dist/ && yarn typechain && tsc && cp temp/*.d.ts dist/ && yarn postbuild",
+    "postbuild": "rm -rf temp/",
+    "typechain": "typechain --target ethers-v5 --out-dir temp/ '../deployments/tasks/*/abi/*.json' 'abi/*.json'"
```

Comment:
This means that we should not use these typings in our tests, correct? As the code in master may have diverged from the last deployment.

Reply:
Yes, I think we shouldn't do that. I'm seeing this as something for Balancer.js to use along with outside developers, trying to keep these up to date for in-development contracts along with maintain backwards compatibility with deployed ones might be tricky.